diff --git a/dist/_browser-chunks/chunk-342N7CWU.js b/dist/_browser-chunks/chunk-342N7CWU.js
index 342f9b9b78c04e71bef8f43a5b6d1a00d14a45c1..48b5c480110b35552a9a4d49046ee9e97630365d 100644
--- a/dist/_browser-chunks/chunk-342N7CWU.js
+++ b/dist/_browser-chunks/chunk-342N7CWU.js
@@ -118,6 +118,8 @@ var getParallelRoutes = (segmentsList) => {
           // @ts-ignore Only available in Next.js >= v15.1.1
           parentCacheNode: newLazyCacheNode,
           // END
+          // Next.js >= v16.4: useRouter reads the bfcache id of the parent render tree.
+          parentRenderTree: { data: { bfcacheId: 0 } },
           url: pathname,
           loading: null
         }
